### PR TITLE
Replace user tracking with Customer Experience Monitoring

### PR DESCRIPTION
**Pull request by izsi-salmon**
[SS-520](https://raygun.atlassian.net/browse/SS-520): Update in app set up instructions for all providers

## Description:

This is a PR to update the product name User Tracking, to the new Customer Experience Monitoring name, as part of the Users 2.0 work.

## Screenshots:
SS-520-wordpress-plugin-updated

## Test plan:
- [ ] New name appears on plugin
- [ ] Formatting stays good
- [ ] Nothing breaks :)

## Author to check:
- [ ] Tested in alternative environment

## Reviewer to check
- [ ] Appropriate tests written
- [ ] Code is written to standards

### CODE REVIEW

**Review by samuel-holt — APPROVED** [review]
These changes look fine - not sure if we should update mention of _user tracking_ [here](https://github.com/MindscapeHQ/raygun4wordpress/blob/b5f3399d7daa8715b518237512adb5126b932220/raygun4wp.php#L5)? I'll leave that up to you :)

File: settings.php
Comment (samuel-holt) [inline settings.php:34]: I would wrap this text with the l10n function as well. 

i.e.

```
<legend class="screen-reader-text"><span><?php _e("Customer Experience Monitoring"); ?></span></legend>
```

**Review by samuel-holt — APPROVED** [review]
Changes all look good. :+1: Just one minor best practice thing I've mentioned

**Comment by izsi-salmon** [comment]
Getting a copy review from product

**Comment by izsi-salmon** [comment]
Product/growth has made the decision to delay releasing the Customer Experience Monitoring product name change in the WordPress plugin until the full release of the new product. This is for privacy/copyright reasons so we can take the time to officially release our new product name, and avoid leaking this rebrand to competitors.

This PR will be blocked until the full product release, when it is approved by product and growth, the PR can be merged and the plugin upload process continued.

**Pull request closed without merging**
